hemtt.toml

```diff
@@ -0,0 +1,26 @@
+name = "Zeus Enhanced"
+prefix = "zen"
+author = "ZEN Team"
+mainprefix = "x"
+sigversion = 3
+version = "1.6.1"
```

Comment:
Don’t use this, makes version numbers a pain because you’d have to update this one too.

It already looks for script_version